website/docs/r/ecs_service.html.markdown

```diff
@@ -80,6 +80,7 @@ The following arguments are supported:
 * `task_definition` - (Required) The family and revision (`family:revision`) or full ARN of the task definition that you want to run in your service.
 * `desired_count` - (Optional) The number of instances of the task definition to place and keep running. Defaults to 0. Do not specify if using the `DAEMON` scheduling strategy.
 * `launch_type` - (Optional) The launch type on which to run your service. The valid values are `EC2` and `FARGATE`. Defaults to `EC2`.
+* `platform_version` - (Optional) The platform version on which to run your service. Defaults to `LATEST`.
```

Comment:
Could you please add a link to the AWS documentation and mention this only applies to the `launch_type` being set to `FARGATE`? e.g.

```suggestion
* `platform_version` - (Optional) The platform version on which to run your service. Only applicable for `launch_type` set to `FARGATE`. Defaults to `LATEST`. More information about Fargate platform versions can be found in the [AWS ECS User Guide](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/platform_versions.html).
```